lib/core/src/Cardano/Wallet/CoinSelection/Internal/Balance.hs

```diff
@@ -187,7 +188,8 @@ import qualified Cardano.Wallet.Primitive.Types.Coin as Coin
 import qualified Cardano.Wallet.Primitive.Types.TokenBundle as TokenBundle
 import qualified Cardano.Wallet.Primitive.Types.TokenMap as TokenMap
 import qualified Cardano.Wallet.Primitive.Types.TokenQuantity as TokenQuantity
-import qualified Cardano.Wallet.Primitive.Types.UTxOIndex as UTxOIndex
+import qualified Cardano.Wallet.Primitive.Types.UTxOIndex.Internal as UTxOIndex
+    ( UTxOIndex (balance), selectRandomWithPriority, toList )
```

Comment:
Hi @paolino 

According to our coding standards, [imports should be grouped into three groups, separated by a blank newline](https://github.com/input-output-hk/adrestia/blob/master/docs/code/Coding-Standards.md#stylish-haskell-is-used-to-format-grouped-imports--language-pragmas):

    Prelude import
    Explicit imports
    Qualified imports

So explicit imports like `UTxOIndex` should be moved to the section above.